jujugui/static/gui/src/app/components/modal-gui-settings/modal-gui-settings.js

```diff
@@ -108,6 +135,7 @@ class ModalGUISettings extends React.Component {
 
 ModalGUISettings.propTypes = {
   closeModal: PropTypes.func.isRequired,
+  flags: PropTypes.object.isRequired,
```

Comment:
was this previously missing? I don't see that it was added in the code above and not sure what flags is here atm.